src/lib/wrap.svelte

```diff
@@ -1,35 +1,31 @@
 <script lang="ts">
- import { createResourceClient, createResourceQuery, useResourceNames } from '@viamrobotics/svelte-sdk';
- import { SensorClient } from '@viamrobotics/sdk';
- import Analyzer from "./analyzer.svelte"
- 
- let {
-   partID,
-   ...rest
- }: Props = $props();
+  import {
+    createResourceClient,
+    createResourceQuery,
+    useResourceNames,
+  } from "@viamrobotics/svelte-sdk";
+  import { ResourceName, SensorClient } from "@viamrobotics/sdk";
+  import Analyzer from "./analyzer.svelte";
 
- let sensorName = $state("all-pgn");
+  let { partID, ...rest }: Props = $props();
 
- function theList() {
-   const resources = useResourceNames(() => partID);
-   const lst = resources.current;
-   lst.sort((a,b) => {
-     return a.name.localeCompare(b.name);
-   });
-   console.log(lst.length);
-   return lst
- }
- 
+  let sensorName = $state("all-pgn");
+
+  const resources = useResourceNames(() => partID);
+  const sorted = $derived(
+    resources.current.toSorted((a, b) => {
+      return a.name.localeCompare(b.name);
+    })
+  );
```

Comment:
This fixes showing the full resource names list in the dropdown - tested locally